<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FTAnalyzerWin</class>
 <widget class="QMainWindow" name="FTAnalyzerWin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1287</width>
    <height>998</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="grpFileList">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>401</width>
      <height>851</height>
     </rect>
    </property>
    <property name="title">
     <string>File List</string>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QTableWidget" name="tabwFiles"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="wdgPreview" native="true">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>90</y>
      <width>851</width>
      <height>581</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>691</width>
      <height>26</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QPushButton" name="btnFTFolder">
       <property name="text">
        <string>Open Folder</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineFilePath"/>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="grpFitResult">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>700</y>
      <width>851</width>
      <height>227</height>
     </rect>
    </property>
    <property name="title">
     <string>Fit Results</string>
    </property>
    <layout class="QGridLayout" name="gridLayout_4">
     <item row="0" column="0">
      <widget class="QTableWidget" name="tabwFitResult"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>30</y>
      <width>311</width>
      <height>26</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="3">
      <widget class="QPushButton" name="btnFit">
       <property name="text">
        <string>Fit</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QSpinBox" name="boxSelectCh">
       <property name="maximum">
        <number>31</number>
       </property>
       <property name="value">
        <number>16</number>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Selected Channel</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1287</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuTest">
    <property name="title">
     <string>Test Menu</string>
    </property>
    <addaction name="actionTest_bar"/>
   </widget>
   <addaction name="menuTest"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionTest_bar">
   <property name="text">
    <string>Test bar</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
